harness/determined/cli/resource_pool.py

```diff
@@ -0,0 +1,166 @@
+
+
+def get_workspaces_string(workspaces: List[str]) -> str:
+    workspaces_str = ""
+    for workspace in workspaces[:-1]:
+        workspaces_str += workspace + ", "
+
+    if workspaces_str != "":
+        workspaces_str += "and "
+    workspaces_str += workspaces[-1]
+    return workspaces_str
+
+
+@authentication.required
+def add_binding(args: Namespace) -> None:
+    body = bindings.v1BindRPToWorkspaceRequest(
+        resourcePoolName=args.pool_name, workspaceNames=args.workspace_names
+    )
+    bindings.post_BindRPToWorkspace(setup_session(args), body=body, resourcePoolName=args.pool_name)
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+
+    print(
+        f'added bindings between the resource pool "{args.pool_name}" '
+        f"and the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def remove_binding(args: Namespace) -> None:
+    body = bindings.v1UnbindRPFromWorkspaceRequest(
+        resourcePoolName=args.pool_name,
+        workspaceNames=args.workspace_names,
+    )
+    bindings.delete_UnbindRPFromWorkspace(
+        setup_session(args), body=body, resourcePoolName=args.pool_name
+    )
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+    print(
+        f'removed bindings between the resource pool "{args.pool_name}'
+        f"and the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def replace_bindings(args: Namespace) -> None:
+    body = bindings.v1OverwriteRPWorkspaceBindingsRequest(
+        resourcePoolName=args.pool_name,
+        workspaceNames=args.workspace_names,
+    )
+    bindings.put_OverwriteRPWorkspaceBindings(
+        setup_session(args), body=body, resourcePoolName=args.pool_name
+    )
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+    print(
+        f'replaced bindings of the resource pool "{args.pool_name}'
+        f"with those to the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def list_workspaces(args: Namespace) -> None:
+    session = setup_session(args)
+    resp = bindings.get_ListWorkspacesBoundToRP(session, resourcePoolName=args.pool_name)
+    if resp.workspaceIds is None or len(resp.workspaceIds) == 0:
+        print("resource pool has no assignments")
+        return
+
+    print("ids are:", resp.workspaceIds)  # TODO: convert to names
+    workspace_names = ""
+    workspaces_set = set(resp.workspaceIds)
+    workspaces = bindings.get_GetWorkspaces(session)
+    for workspace in workspaces.workspaces:
+        if workspace.id in workspaces_set:
+            workspace_names += workspace.name + ", "
+
+    if workspace_names != "":
+        workspace_names = workspace_names[:-2]
+
+    headers = ["resource pool", "workspaces"]
+    values = [[args.pool_name, workspace_names]]
+    render.tabulate_or_csv(headers, values, False)
+    return
+
+
+args_description = [
```

Comment:
I continue to like both how naming came together for users of `det` and how you've worked it into function names within the module.